docs/metrics/metrics-operators/filter.md

```diff
@@ -4,10 +4,13 @@ title: filter Metrics Operator
 sidebar_label: filter
 ---
 
+:::note
+The functionality provided by the `filter` operator has been incorporated into the `where` operator. We recommend the use of `where` over `filter`, because `filter` will be deprecated in the future. For more information see [where Metrics Operator](docs/metrics/metrics-operators/where.md)
```

Comment:
```suggestion
The functionality provided by the `filter` operator has been incorporated into the `where` operator. We recommend the use of `where` over `filter`, because `filter` will be deprecated in the future. For more information, see [where Metrics Operator](docs/metrics/metrics-operators/where.md)
```